### PR TITLE
[100.0.0+up1.10.4] Add kube-version annotations to istio 1.10

### PR DESCRIPTION
**Pull request by doflamingo721**
Linked issue:
https://github.com/rancher/rancher/issues/37697

### CODE REVIEW

File: packages/rancher-istio/rancher-tracing/charts/templates/_helpers.tpl [hunk 1]
```diff
@@ -30,18 +30,3 @@ If release name contains chart name it will be used as a full name.
 {{- end -}}
 {{- end -}}
 {{- end -}}
-
```
Comment (aiyengar2) [inline packages/rancher-istio/rancher-tracing/charts/templates/_helpers.tpl:33]: why are we dropping these changes?

File: packages/rancher-istio/rancher-tracing/charts/templates/deployment.yaml [hunk 2]
```diff
@@ -74,14 +74,6 @@ spec:
         runAsUser: 1000
       serviceAccountName: {{ include "tracing.fullname" . }}
       {{- end }}
-      nodeSelector: {{ include "linux-node-selector" . | nindent 8 }}
```
Comment (aiyengar2) [inline packages/rancher-istio/rancher-tracing/charts/templates/deployment.yaml:77]: why are we dropping these changes

File: packages/rancher-istio/rancher-tracing/charts/values.yaml [hunk 3]
```diff
@@ -1,11 +1,6 @@
 provider: jaeger
 contextPath: ""
-## Node labels for pod assignment
```
Comment (aiyengar2) [inline packages/rancher-istio/rancher-tracing/charts/values.yaml:3]: why are we dropping these changes?

**Review by aiyengar2 — CHANGES_REQUESTED** [review]
Currently, this PR is dropping additional changes we added for Windows support.

Also see comment on https://github.com/rancher/charts/pull/1841#discussion_r875425618 about multiple release lines.

**Comment by doflamingo721** [comment]
Closing this PR in favour of https://github.com/rancher/charts/pull/1905

**Pull request closed without merging**